File: Src/WebApiServer/DataAccessLayer/Models/Entities/WalletModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataAccessLayer.Models.Entities
{
    public class WalletModel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public decimal Amount { get; set; }
        public string CurrencyCode { get; set; }
        public int IdUser { get; set; }
        public UserModel? User { get; set; }
        public ICollection<TransactionModel>? Transactions { get; set; }
    }
}

File: Src/WebClientApp/WebClient/src/app/_models/Wallet/walletModel.ts
export class WalletModel {
	id: number | undefined;
	name: string;
	amount: number;
	currencyCode: string;

	constructor(name = '', amount = 0, currencyCode = '') {
		this.name = name;
		this.amount = amount;
		this.currencyCode = currencyCode;
	}
}
